openedx/features/content_tagging/rules.py

```diff
@@ -35,27 +37,51 @@ def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool
     return False
 
 
+def is_taxonomy_admin(user: User) -> bool:
+    """
+    Returns True if the given user is a Taxonomy Admin.
+
+    Taxonomy Admins include global staff and superusers.
+    """
+    return oel_tagging.is_taxonomy_admin(user)
+
+
 @rules.predicate
 def can_view_taxonomy(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
     """
-    Anyone can view an enabled taxonomy,
-    but only taxonomy admins can view a disabled taxonomy.
+    Everyone can potentially view a taxonomy (taxonomy=None). The object permission must be checked
+    to determine if the user can view a specific taxonomy.
+    Only taxonomy admins can view a disabled taxonomy.
     """
-    if taxonomy:
-        taxonomy = taxonomy.cast()
-    return (taxonomy and taxonomy.enabled) or is_taxonomy_admin(user, taxonomy)
+    if not taxonomy:
+        return True
+
+    taxonomy = taxonomy.cast()
+
+    return taxonomy.enabled or is_taxonomy_admin(user)
+
+
+@rules.predicate
+def can_add_taxonomy(user: User) -> bool:
+    """
+    Only taxonomy admins can add taxonomies.
+
+    To do: There is currently no REST API method to create a taxonomy associated with an organization
+    neither to add a taxonomy to an organization.
```

Comment:
Please remove these lines -- These REST API methods are part of implementing the frontend requirements, and so we won't forget to add them.

```suggestion
```